src/components/FloatingHelper/FloatingHelper.tsx

```diff
@@ -34,10 +34,8 @@ export type FloatingHelperProps = PickedClosablePopoverProps & FloatingHelperOwn
 
 export const FloatingHelper: React.SFC<FloatingHelperProps> = props => {
   const { children, width, content, showCloseButton, ...rest } = props;
-  const contentWidth = (typeof width) === 'number' ? `${width}px` : width;
```

Comment:
This was remove since it is not really needed. I was confusing with the case where we pass a string which is a number representation, like "600"